lxd/api_cluster.go

```diff
@@ -546,10 +546,10 @@ func clusterPutJoin(d *Daemon, r *http.Request, req api.ClusterPut) response.Res
 	run := func(op *operations.Operation) error {
 		logger.Debug("Running cluster join operation")
 
-		// If the user has provided a cluster password, setup the trust
+		// If the user has provided a join token, setup the trust
```

Comment:
The commit message is "lxd: Remove core.https_address " but this isnt what the change is about.

Reply:
Fixed. The commit is now split according to the files packages.